### PR TITLE
wrapping up; fixed repeated keys in rxn_gene_complex, added KEGG gene …

### PR DESCRIPTION
**Pull request by sean-mccorkle**
…model handling, added bash directory loader

### CODE REVIEW

File: src/utils/shell_load_directory.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+#
+#  make sure DB_USER, DB_PASS and DB_URL are properly set as well
+#  as BINDIR. then
+#
+#    python shell_load_directoy.py  JSONdirectory
```
Comment (jayrbolton) [inline src/utils/shell_load_directory.sh:6]: do you mean `bash shell_load_directory.sh <dir>`?

File: src/KEGG/generate_kegg_complexes.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+                                ########
+                                # Main #
+                                ########
+
+kegg_mapping_file = sys.argv[1]
+output_dir = sys.argv[2]
+
+read_kegg_mapping( kegg_mapping_file )
```
Comment (jayrbolton) [inline src/KEGG/generate_kegg_complexes.py:65]: Wrap all these function calls in:

```
if __name__ == '__main__':
    # function calls
```
That way, if we ever want to import this module, eg into the py shell, then the functions won't run on import

File: src/utils/shell_load_directory.sh (hunk 1 above)
Comment (sean-mccorkle) [reply src/utils/shell_load_directory.sh]: Yep - just pushed the fix

File: src/KEGG/generate_kegg_complexes.py (hunk 2 above)
Comment (sean-mccorkle) [reply src/KEGG/generate_kegg_complexes.py:65]: okay

Comment (sean-mccorkle) [reply src/KEGG/generate_kegg_complexes.py]: just pushed that fix

**Pull request merged**